### PR TITLE
Add routes and controllers for courier

### PR DESCRIPTION
**Pull request by ezecodes**

### CODE REVIEW

File: src/controller/auth.controller.js
Comment (Lekejosh) [inline src/controller/auth.controller.js:1]: Why adding role to login controller?

All users have there roles attached to them in the DB... No need to specifiy again... Fix your eslint errors

Did you disable husky on your end?

File: seeders/20240122220926-demo-flow.js
Comment (Lekejosh) [inline seeders/20240122220926-demo-flow.js:1]: Fix duplicate email errors here

File: src/controller/Order/order.controller.js
Comment (Lekejosh) [inline src/controller/Order/order.controller.js:1]: Fix eslint errors here

File: src/controller/courier/order.controller.js
Comment (Lekejosh) [inline src/controller/courier/order.controller.js:1]: fix the unneccessary semicolon errors in all the files in your PR

**Review by Lekejosh — CHANGES_REQUESTED** [review]
Fix all the errors and check the comments made

**Comment by jahdevelops** [comment]
Fix: Remove role auth from login route

**Pull request merged**